cns/service/main.go

```diff
@@ -813,7 +813,7 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 	}
 
 	// initialize the ipam pool monitor
-	httpRestServiceImplementation.IPAMPoolMonitor = ipampoolmonitor.NewCNSIPAMPoolMonitor(httpRestServiceImplementation, requestController)
+	httpRestServiceImplementation.IPAMPoolMonitor = ipampoolmonitor.NewPoolMon(httpRestServiceImplementation, requestController)
```

Comment:
nit: Can we rename it to NewAutoScalerMonitor (NewPoolMon is too generic, IMO)

Reply:
It can't be used without its package qualifier `ipampoolmonitor.` so it's already unambiguous. It's arguably better Go style to go the other way and make it just `ipampool.Monitor`.
See[ `http.Client`](https://pkg.go.dev/net/http#Client) for similar convention.